tf-modules/cumulus/data-migrations.tf

```diff
@@ -16,21 +16,3 @@ module "data_migration2" {
 
   tags = var.tags
 }
-
-module "data_migration2_ecs_service" {
```

Comment:
This is removed because it should have been removed when we updated in favor of running it as an async operation?

Reply:
Indeed